demisto_sdk/commands/format/format_module.py

```diff
@@ -346,10 +358,10 @@ def run_format_on_file(
             FileType.LAYOUT.value,
             FileType.MAPPER.value,
         )
-        and "id_set_path" in kwargs
+        and "graph" in kwargs
     ):
         # relevant only for incidentfield/layouts/mappers
-        del kwargs["id_set_path"]
+        del kwargs["graph"]
```

Comment:
do we still want to delete it? Don't we need the graph for these content item types?

Reply:
We need the graph only for mappers, layout and incident fields. For the rest of the content items types we don't need it and can delete it. In this case (when it's not the 3 type I mentioned), the value is None anyway, so we can delete it.